src/Core/src/Platform/iOS/SearchBarExtensions.cs

```diff
@@ -1,10 +1,29 @@
-﻿using Foundation;
+using System;
+using Foundation;
+using Microsoft.Maui.Graphics;
 using UIKit;
 
 namespace Microsoft.Maui.Platform
 {
 	public static class SearchBarExtensions
 	{
+		// TODO: NET7 maybe make this public?
+		internal static void UpdateBackground(this UISearchBar uiSearchBar, ISearchBar searchBar)
+		{
+			uiSearchBar.UpdateBackground(searchBar, null);
+		}
+
+		// TODO: NET7 maybe make this public?
+		internal static void UpdateBackground(this UISearchBar uiSearchBar, ISearchBar searchBar, UIColor? defaultTintColor = null)
+		{
+			ViewExtensions.UpdateBackground(uiSearchBar, searchBar);
+
+			var background = searchBar.Background;
+
+			if (background is SolidPaint solidPaint)
+				uiSearchBar.BarTintColor = solidPaint.Color.ToPlatform(defaultTintColor ?? ColorExtensions.BackgroundColor);
+		}
```

Comment:
Is this the correct way to change colors? How would one go about changing the text box background color? Or are they all related. What if the user wants to have the "background" a color or transparent, and the search box a different color? Should we be doing a minimal style?

Reply:
What do we do if the color is null?